src/main/resources/db/migrationv2/V2_10__certificate_of_attendance_performance.sql

```diff
@@ -0,0 +1,8 @@
+drop index juror_history_history_code_idx;
+CREATE INDEX juror_history_history_code_idx ON juror_mod.juror_history USING btree (history_code, juror_number);
```

Comment:
should we do this as a script after the release deploys - as a separate script that can take as much time as required - just leave the drop statements in here